code/game/objects/items/weapons/grenades/chronogrenade.dm

```diff
@@ -37,14 +37,23 @@
 
 	playsound(A, 'sound/effects/fall.ogg', 100, 0, 0, 0, 0)
 
-	for(var/turf/T in targets)
-		for(var/atom/movable/everything in T)
-			if(everything.flags & TIMELESS || everything.being_sent_to_past)	//allowing future grenades to interact with past-tethered atoms would be a nightmare
-				continue
-			everything.send_to_future(duration)
-			if(ismob(everything))
-				var/mob/M = everything
-				M.playsound_local(everything, 'sound/effects/fall2.ogg', 100, 0, 0, 0, 0)
+	if(single_target)
+		if(!A.being_sent_to_past)
```

Comment:
So a single target can be affected even if timeless? Also, there's a lot of copypaste to all of this - make these procs.